convert2rhel/actions/system_checks/is_loaded_kernel_latest.py

```diff
@@ -0,0 +1,171 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+import os
+
+from convert2rhel import actions
+from convert2rhel.pkghandler import compare_package_versions
+from convert2rhel.repo import get_hardcoded_repofiles_dir
+from convert2rhel.systeminfo import system_info
+from convert2rhel.utils import run_subprocess
+
+
+logger = logging.getLogger(__name__)
+
+
+class IsLoadedKernelLatest(actions.Action):
```

Comment:
There is an open PR (https://github.com/oamg/convert2rhel/pull/817) that do some changes to the old code in checks.py. 

If the #817 is merged first, we need to take care of this here, otherwise, we should take care of updating #817 for the contributor.